FordLabs/PeopleMover
# Sort dropdowns

## Issue
Connects #355  

## What was done
- [x] Dropdowns are now sorted in person, regardless of uppercase or lowercase

## How to test
1. Go to the add a person modal and see if the assing to and role dropdowns are sorted
